addons/website/static/src/js/menu/translate.js

```diff
@@ -178,6 +178,36 @@ var TranslatePageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             devicePreview: false,
         };
 
+        const showTip = ev => {
+            const $target = $(ev.target);
+            let message = _t('This translation is not editable.');
+            if (ev.target.closest('.s_table_of_content_navbar_wrap')) {
+                message = _t('Translate header in the text. Menu is generated automatically.');
+            }
+            // Tooltip appearance and disappearance should not make the translation dirty.
+            this.translator.wysiwyg.odooEditor.observerUnactive();
+            $target.tooltip({
+                title: message,
+                trigger: 'hover',
+                container: 'body',
+                delay: {'hide': 800},
+            }).on('inserted.bs.tooltip', ev => {
+                this.translator.wysiwyg.odooEditor.observerActive();
+            }).on('show.bs.tooltip', ev => {
+                this.translator.wysiwyg.odooEditor.observerUnactive();
+            }).on('shown.bs.tooltip', ev => {
+                this.translator.wysiwyg.odooEditor.observerActive();
+            }).on('hide.bs.tooltip', ev => {
+                this.translator.wysiwyg.odooEditor.observerUnactive();
+            }).on('hidden.bs.tooltip', ev => {
+                $target.tooltip('dispose');
+                this.translator.wysiwyg.odooEditor.observerActive();
+            }).tooltip('show');
```

Comment:
That looks too fragile, despite probably working correctly IMHO. Especially knowing how the editor behaves with those observer things.
It looks like an open door for issues (both because of the tooltip mechanism and the editor).

Wouldn't we be better with a toaster/notification on click on this element? Would be a bit less good UX (maybe not actually) but more straightforward code and less prone to future error.

Note that I see a simpler form of what is done here there: https://github.com/odoo/odoo/blame/f1f3fcef6cc471fc01da574da26712e643315da6/addons/web_editor/static/src/js/wysiwyg/wysiwyg.js#L1782-L1784
Doesn't seems enough tho, I asked: https://github.com/odoo/odoo/pull/71168/files#r1008253534

Reply:
Checked with PDE, he is fine with a notification - it is therefore a notification now.